Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -69,6 +70,23 @@ final class ContentRepository
      */
     private array $projectionStateCache;
 
+    /**
+     * @var array<string, ContentGraphInterface> Runtime cache for {@see ContentGraphInterface} instances, indexed by their workspace name
+     */
+    private array $contentGraphInstancesByWorkspaceName = [];
+
+    /**
+     * @var array<string, Workspace> Runtime cache for {@see Workspace} instances, indexed by their name
+     */
+    private array $workspaceInstancesByName = [];
+
+    /**
+     * @var array<string, ContentStream> Runtime cache for {@see ContentStream} instances, indexed by their name
+     */
+    private array $contentStreamInstancesById = [];
```

Comment:
Are those new runtime caches related? I would prefer to leave them for a separate PR

Reply:
removed